tools/ci_connector_ops/ci_connector_ops/pipelines/commands/airbyte_ci.py

```diff
@@ -13,12 +15,34 @@
     get_current_git_revision,
     get_modified_files_in_branch,
     get_modified_files_in_commit,
+    get_modified_files_in_pull_request,
 )
+from github import PullRequest
 
 from .groups.connectors import connectors
 from .groups.metadata import metadata
 
 
+def get_modified_files(
+    git_branch: str, git_revision: str, diffed_branch: str, is_local: bool, ci_context: CIContext, pull_request: PullRequest
+) -> List[str]:
+    """Get the list of modified files in the current git branch.
+    If the current branch is master, it will return the list of modified files in the current commit.
```

Comment:
Suggestion: Maybe clarify that this "current commit" is the squash commit of the branch whose merge kicked off the run

The reason I mention this is that if we decided to switch from squash commits to a full history (e.g. rebase instead of merge), I'm not sure if this "last commit" logic would work anymore. Its an edge case not worth writing more code for, but a reminder of the context that allows us to do it this way may be helpful.

Reply:
I improved the docstring, thank you for the suggestion.